src/Command/ScanCommand.php

```diff
@@ -104,7 +118,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $scanner = new Scanner($dispatcher, new CallbackVisitor($ruleFactory->createRuleCollection()));
 
         $scanner->scan(
-            new FileIterator($input->getArgument('path'))
+            new FileIterator(
+                $input->getArgument('path'),
+                array_filter(explode(',', $input->getOption('ignore-paths'))),
```

Comment:
I'm not sure why you're using `array_filter()` here. Are you expecting it to remove empty values? If so, that's fine, but it's a little obscure. I didn't realize it was usable like that until I just looked it up. I'd be tempted to refactor that bit  (`array_filter(explode(...))`) into a separate method and document this use of array_filter(). But also see my other comments.